packages/frontend/helpers/assets.ts

```diff
@@ -291,3 +295,33 @@ export const withdrawMaxAmount = async (
 
   return new FujiResultSuccess(amount);
 };
+
+export const invalidBridgingAmount = (
+  steps: RoutingStepDetails[],
+  collateral: AssetChange,
+  debt?: AssetChange
+) => {
+  for (const step of steps) {
+    if (step.step === RoutingStep.X_TRANSFER && step.token) {
+      const asset = [collateral, debt]
+        .filter((asset) => asset)
```

Comment:
what's the point of this filter

Reply:
First, we want only `X_TRANSFER` steps since the `steps` array contains all of them. Then we make sure it has a token since its optional and finall grab the asset since it already contains price in USD